development/build/manifest.js

```diff
@@ -47,8 +77,9 @@ function createManifestTasks({
           browserVersionMap[platform],
           await getBuildModifications(buildType, platform),
           customArrayMerge,
+          // Only include _flags if manifestFlags has content
+          manifestFlags && { _flags: manifestFlags },
```

Comment:
```suggestion
          manifestFlags,
```

Reply:
Thanks for the suggestions ! It is indeed more generic to not include `flags` in. addressed ab7448dc61790c45092eb55b173ac86815618ec2